# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1510108378 18000
#      Tue Nov 07 21:32:58 2017 -0500
# Node ID 68c4ee9545b2d52ddf2f33e5638f2f97275ff44e
# Parent  014f470896826a8dbd6ae87898951534c75a6a11
* Figure.cc: Fix typo from previous change.

diff --git a/libgui/graphics/Figure.cc b/libgui/graphics/Figure.cc
--- a/libgui/graphics/Figure.cc
+++ b/libgui/graphics/Figure.cc
@@ -872,17 +872,17 @@ namespace QtHandles
 
     if (prompt || file.isEmpty ())
       {
         QFileInfo finfo (file);
 
         file = QFileDialog::getSaveFileName (qWidget<FigureWindow> (),
                                              tr ("Save Figure As"),
                                              finfo.absoluteFilePath (),
-                                             tr ("Figure File (*.fig);;Vector Image Formats (*.eps *.epsc *.pdf *.svg *.ps *.tikz);;Bitmap Image Formats (*.gif *.jpg *.png *.tiff)",
+                                             tr ("Figure File (*.fig);;Vector Image Formats (*.eps *.epsc *.pdf *.svg *.ps *.tikz);;Bitmap Image Formats (*.gif *.jpg *.png *.tiff)"),
                                              nullptr,
                                              QFileDialog::DontUseNativeDialog);
     }
 
     if (! file.isEmpty ())
       {
         QFileInfo finfo (file);
 
